### DISCUSSION TITLE
Array as value of bundle argument

### DISCUSSION THREAD

**Discussion by kiriencoder** [Permify/permify#1261] [Q&A]
Can I use an array of IDs as argument value in a bundle?
For example, I'm creating bundle to delete list of projects:
"bundles": [
    {
        "name": "projects-delete"
        "arguments": [
            "projectID",
            "ownerIDs",
        ],
        "operations": [
            {
                "relationships_delete": [
                    "project:{{.projectID}}#owner@user:{{.ownerIDs}}",
                ],
            },
        ],
    },
],
and when I send list of ownerIDs ['1','2','3','4'], the bundle will delete next relations:
project:1:owner#user@1
project:1:owner#user@2
project:1:owner#user@3
project:1:owner#user@4

**Comment by kahyoongho** [comment]
seems like bundle dont allow array as argument value

**Reply by Charlotte1959** [reply]
Arrayed
